### PR TITLE
Add endpoint to check if OC attach mail can be filed.

### PR DESCRIPTION
**Pull request by lukasgraf**
Add endpoint to check if OC attach mail can be filed.

For [CA-5662](https://4teamwork.atlassian.net/browse/CA-5662)

## Checklist

- [x] Changelog entry
- [x] Link to issue (Jira or GitHub) and backlink in issue (Jira)

- API change:
  - [ ] Documentation is updated
  - [x] API Changelog entry

[CA-5662]: https://4teamwork.atlassian.net/browse/CA-5662?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: opengever/officeconnector/testing.py [hunk 1]
```diff
@@ -105,6 +105,26 @@ def fetch_dossier_multiattach_oc_url(self, browser, dossier, documents, bcc):
 
         return url
 
+    def fetch_oc_attach_is_mail_fileable(self, browser, dossier, documents):
```
Comment (njohner) [inline opengever/officeconnector/testing.py:108]: I don't think you need the `dossier` parameter?

File: opengever/officeconnector/tests/test_api_is_mail_fileable.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
+from ftw.testbrowser import browsing
+from opengever.dossier.resolve import LockingResolveManager
+from opengever.officeconnector.testing import OCSolrIntegrationTestCase
+
+
+class TestOfficeconnectorIsMailFileable(OCSolrIntegrationTestCase):
+
+    features = ("!officeconnector-checkout", "officeconnector-attach")
```
Comment (njohner) [inline opengever/officeconnector/tests/test_api_is_mail_fileable.py:8]: Why the `"!officeconnector-checkout"`?

**Review by njohner — APPROVED** [review]
LGTM, just some comments regarding testing.

File: opengever/officeconnector/tests/test_api_is_mail_fileable.py (hunk 2 above)
Comment (lukasgraf) [reply opengever/officeconnector/tests/test_api_is_mail_fileable.py]: Thanks, copy paste fail from another test I based this on.

**Pull request merged**